packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -512,4 +658,86 @@ contract EpochManager is
       CELOequivalent
     );
   }
+
+  /**
+   * @notice Updates the list of elected validator signers.
+   */
+  function _setElectedSigners(address[] memory _elected) internal {
+    IAccounts accounts = getAccounts();
+    require(electedAccounts.length > 0, "Elected list length cannot be zero.");
+    if (electedSigners.length == 0) {
+      electedSigners = new address[](_elected.length);
+    }
+
+    require(_elected.length == electedSigners.length, "Account and Signer list lenght mismatch.");
```

Comment:
we cannot check whether _elected.length == electedSigners.length since in scenario when previously we had elected eg only 10 validators (because there were only 10 validators registered) and during this epoch new validators registered then it is valid to have different count of electedSigners from before

Reply:
I saw this conversation marked as resolved, I think the issue still remains